setup.py

```diff
@@ -1,6 +1,9 @@
 from setuptools import setup, find_packages
 
 
+VERSION = "21.1.0.dev0"
```

Comment:
This doesn't actually get used anywhere right? Is there a reason for setting it as a variable in setup.py? (as opposed to it's own version file that only contains the version information or similar)

Reply:
Just mimicked what Sentry did. Maybe in the future we can use this in `snuba --help` or something?

Reply:
Ok